src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
```

Comment:
and then use it here, to avoid writing strings, also `public` modifier is redundant, all fields are public by default, fix all of these everywhere
```suggestion
  shape: Shape = Shape.Triangle;
```